endpoints/getting-started-grpc/README.md

```diff
@@ -41,14 +41,14 @@ Cloud account and [SDK](https://cloud.google.com/sdk/) configured.
 1. Deploy your service config to Service Management:
 
     ```bash
-    gcloud service-management deploy out.pb api_config.yaml
+    gcloud endpoints services deploy out.pb api_config.yaml
     # The Config ID should be printed out, looks like: 2017-02-01r0, remember this
 
     # set your project to make commands easier
     GOOGLE_CLOUD_PROJECT=<Your Project ID>
 
     # Print out your Config ID again, in case you missed it
-    gcloud service-management configs list --service hellogrpc.endpoints.${GOOGLE_CLOUD_PROJECT}.cloud.goog
+    gcloud endpoints services configs list --service hellogrpc.endpoints.${GOOGLE_CLOUD_PROJECT}.cloud.goog
```

Comment:
Should you remove this config ID stuff? (and above, where it says "config ID should be printed out, looks like ...)

Reply:
Config Id - yes, but this command is actually helpful to also get the service name, so I prefer to leave it - just change the documentation slightly.